script.js

```diff
@@ -1,31 +1,47 @@
 const state = {
+  shows: [],
   episodes: [],
   searchTerm: "",
+  selectedShowId: "",
   selectedEpisodeId: "",
   error: null,
 };
 
+async function fetchShows() {
+  try {
+    const response = await fetch("https://api.tvmaze.com/shows");
+    const data = await response.json();
+    state.shows = data;
+
+    state.shows.sort(function (a, b) {
+      return a.name.toLowerCase().localeCompare(b.name.toLowerCase());
```

Comment:
THis is a really nice touch but it is a really obscure function. It's the easiest way of doing what you need to do so that isn't bad but I would recommend poping a comment explaining it next time you use an obscure function